strings-csharp/StringFormattableString/StringFormattableString/FormattableStringsMethods.cs

```diff
@@ -8,22 +9,45 @@ namespace StringFormattableString
     [Orderer(SummaryOrderPolicy.FastestToSlowest)]
     public class FormattableStringsMethods
     {
-        [Benchmark]
-        public string StringExample() 
+        public IEnumerable<object> StringValues() 
         {
-            var studentName = "John";
-            var studentAge = 30;
+            yield return new object [] { "John", 30 };
+        }
 
+        [Benchmark]
+        [ArgumentsSource(nameof(StringValues))]
+        public string StringExample(string studentName, int studentAge) 
+        {
             return $"My name is {studentName} and I am {studentAge} years old.";
         }
 
         [Benchmark]
-        public FormattableString FormattableStringExample() 
+        [ArgumentsSource(nameof(StringValues))]
+        public FormattableString FormattableStringExample(string studentName, int studentAge) 
         {
-            var studentName = "Sean";
-            var studentAge = 40;
-            
             return $"My name is {studentName} and I am {studentAge} years old.";
         }
+
+        public FormattableString FormattableSQLStringExample(string userName)
+        {
+            return $"SELECT * FROM Users WHERE Username = {userName}"; 
+        }
```

Comment:
The username string is not quoted properly, it should be `'{userName}'`, but  that being said I would highly recommend removing this example as it is bad practice. This code leads to SQL injection attacks. The statement should be written:
`"SELECT * FROM Users WHERE Username = @UserName"`
And then when it is used we add the parameter value like this:
`cmd.Parameters.Add(new SqlParameter("@UserName", userName));`

Reply:
@martinchege324 - This one still has not been resolved. The example should really just be removed as it is bad practice.